src/app/forms/custom-sections/dimensions/dimensions.component.ts

```diff
@@ -24,7 +24,7 @@ export class DimensionsComponent implements OnInit, OnChanges, OnDestroy {
   constructor(private dfs: DynamicFormService) {}
 
   ngOnInit(): void {
-    this.form = this.dfs.createForm(this.template, this.vehicleTechRecord) as CustomFormGroup;
+    this.form = this.template ? (this.dfs.createForm(this.template, this.vehicleTechRecord) as CustomFormGroup) : ({} as CustomFormGroup);
```

Comment:
Why is this check being applied now?

The new vehicle types do not use these custom sections, so they won't render on the page anyway.

Likewise, for the other 2 custom sections where you made the same change.